frontend_lib/src/component/DropdownMenu/DropdownMenu.jsx

```diff
@@ -33,7 +33,7 @@ const DropdownMenu = props => {
         className={classnames('dropdownMenu dropdown-menu', props.menuCustomClass)}
       >
         {(props.children.length > 1
-          ? props.children.map(child => child && <DropdownMenuItem key={child.childrenKey} customClass={props.itemCustomClass}> {child} </DropdownMenuItem>)
+          ? props.children.map(child => child && <DropdownMenuItem key={child.childkey} customClass={props.itemCustomClass}> {child} </DropdownMenuItem>)
```

Comment:
See https://github.com/tracim/tracim/issues/3698 if you want to handle this now.

A way to get rid of this issue could be to remove prop `childkey` from `child` before using as a child of `DropdownMenuItem`. Maybe something like `{...child, childrenKey: undefined}`.

Reply:
I didn't know that we had an issue for this, I'll go back to the original version. Thx

Reply:
Yes, I did an issue because when I saw it, it was already merged.
I should have mentioned it to you

Reply:
Your childKey key was fine (better than childrenKey?). The problem is that childKey is added to child and React complains because there is no HTML attribute named childKey on this element.